benefit/data/sheets/CaseTypeTab/CaseTabType-sandl-nonprod.json

```diff
@@ -14,5 +14,7 @@
     {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "listing_requirements_superuser", "TabLabel": "Listing Requirements", "TabDisplayOrder": 24, "CaseFieldID": "hearingState", "TabFieldDisplayOrder": 9},
     {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "listing_requirements_superuser", "TabLabel": "Listing Requirements", "TabDisplayOrder": 24, "CaseFieldID": "defaultOverrideFields", "TabFieldDisplayOrder": 10},
     {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "listing_requirements_superuser", "TabLabel": "Listing Requirements", "TabDisplayOrder": 24, "CaseFieldID": "overrideFields", "TabFieldDisplayOrder": 11},
-    {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "listing_requirements_superuser", "TabLabel": "Listing Requirements", "TabDisplayOrder": 24, "CaseFieldID": "amendReasons", "TabFieldDisplayOrder": 12, "FieldShowCondition": "[STATE]=\"readyToList\""}
-]
+    {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "listing_requirements_superuser", "TabLabel": "Listing Requirements", "TabDisplayOrder": 24, "CaseFieldID": "amendReasons", "TabFieldDisplayOrder": 12, "FieldShowCondition": "[STATE]=\"readyToList\""},
+    {"LiveFrom": "", "CaseTypeID": "Benefit","TabID": "outcome_superuser", "TabLabel": "Outcome", "TabDisplayOrder": 30, "CaseFieldID": "caseOutcome", "TabFieldDisplayOrder": 1},
+    {"LiveFrom": "", "CaseTypeID": "Benefit", "TabID": "outcome_superuser", "TabLabel": "Outcome", "TabDisplayOrder": 30, "CaseFieldID": "didPoAttend", "TabFieldDisplayOrder": 2}
```

Comment:
Has something gone missing from here? Perhaps another story but we've had bugs/changes raised where only super users could see/action certain things. Do we need more outcome entries for different roles?

Reply:
I know that I have not been in this file, But do you want outcomes for judge, clerk and registrar in the same fashion as the super user?